source/documentation/concepts/migrate-from-td.md

```diff
@@ -0,0 +1,118 @@
+### Migrade from Template Deploy
```

Comment:
All headings need to be promoted one level higher.
![Screen Shot 2019-09-19 at 10 00 18](https://user-images.githubusercontent.com/2338/65229536-55d55400-dac4-11e9-9069-84ddef9ba646.png)